src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_evp.c

```diff
@@ -93,6 +93,30 @@ int32_t CryptoNative_EvpDigestCurrent(const EVP_MD_CTX* ctx, uint8_t* md, uint32
     return 0;
 }
 
+int32_t CryptoNative_EvpDigestOneShot(const EVP_MD* type, const void* source, int32_t sourceSize, uint8_t* md, uint32_t* mdSize)
+{
+    if (type == NULL || sourceSize < 0 || md == NULL || mdSize == NULL)
+        return 0;
```

Comment:
This reads somewhant "unnatural", as `0` is commonly associated with success.
Can you use a constant instead? 

If this is obvious, so please ignore the comment.

Reply:
OpenSSL (and therefore the Linux crypto shim) uses 0(fail)/1(success).

Reply:
Oh, thanks for the info!